erpnext/projects/doctype/task/task.js

```diff
@@ -49,7 +49,11 @@ frappe.ui.form.on("Task", {
 			},
 			callback: function (r) {
 				if (r.message.length > 0) {
-					frappe.msgprint(__(`Cannot convert it to non-group. The following child Tasks exist: ${r.message.join(", ")}.`));
+					let message = __(
+						'Cannot convert Task to non-group because the following child Tasks exist: {0}.',
```

Comment:
```suggestion
					let message = __('Cannot convert Task to non-group because the following child Tasks exist: {0}.',
```
Text should start on the same line for parser to detect.